FirebaseAppCheck/Tests/Unit/Swift/AppCheckAPITests.swift

```diff
@@ -170,20 +170,20 @@ final class AppCheckAPITests {
             // ...
           }
         }
-      }
```

Comment:
Was this intended to close here and add an extra indentation level?

Reply:
Yes, it fixes an Xcode 13 build beta 2 (and beta 3) build issue. See #8402